lib/iris/analysis/_grid_angles.py

```diff
@@ -24,6 +24,7 @@
 
 import numpy as np
 
+import cartopy.crs as ccrs
 import iris
 
 
```

Comment:
Potential changes to the `_3d_xyz_from_latlon` function
```
    Returns:
        xyz : (array, dtype=float64)
            cartesian coordinates on a unit sphere.  Dimension 0 maps x,y,z.
```
Clarify the last sentence? `Dimension 0 maps x,y,z`

`lon1` and `lat1` -> `lon_rad` and `lat_rad`? seems a little more informative